src/plugins/stringStyle.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import createRule from '../utils/createRule'
+import type {JssStyle, RuleOptions, Rule} from '../types'
+
+const nlOrSemi = /\n|;/
+
+// This is a very optimistic and simplified implementation.
+// It is not intended to implement a full CSS parser here.
+const toStyle = (cssText) => {
+  const style = {}
+  const split = cssText.trim().split(nlOrSemi)
+  for (let i = 0; i < split.length; i++) {
+    const item = split[i]
+    if (!item) continue
+    const colonIndex = item.indexOf(':')
+    // Missing colon, must by a syntax error.
```

Comment:
You could also do another `indexOf(' ')` to auto insert the missing semicolon in effect monkey patching the syntax error.